classes/Cards/hand.py

```diff
@@ -9,12 +9,25 @@ def cards_to_play(card):
 
 # place holder for discarded cards
 class DiscardPile:
-    def add_to_discard_pile(card):
-        pass
+    def __init__(self) -> None:
+        self._discard_pile = []
+
+    def add_to_discard_pile(self, card):
+        self._discard_pile.append(card)
+
+    def send_to_deck(self):
+        self._return_pile = []
```

Comment:
It looks like "_return_pile" is just used in the function. That means it shouldn't need to be a part of the class "self."

You could drop the self. in front of it. But I don't think its a big deal.